src/irr/asset/CGLSLBRDFBuiltinIncludeLoader.h

```diff
@@ -641,13 +774,170 @@ vec3 irr_glsl_ggx_height_correlated_cos_eval(in irr_glsl_BSDFIsotropicParams par
 #include <irr/builtin/glsl/bsdf/brdf/specular/geom/smith.glsl>
 #include <irr/builtin/glsl/bsdf/brdf/specular/fresnel/fresnel.glsl>
 
+//i wonder where i got irr_glsl_ggx_smith_height_correlated() from because it looks very different from 1/(1+L_v+L_l) form
```

Comment:
bu juggling the denominator around and moving the 1.0 in the front into the square roots?